exercises/concept/black-jack/.meta/exemplar.py

```diff
@@ -1,68 +1,93 @@
+"""Functions to help play and score a game of blackjack.
+
+How to play blackjack:    https://bicyclecards.com/how-to-play/blackjack/
+"Standard" playing cards: https://en.wikipedia.org/wiki/Standard_52-card_deck
+
+"""
```

Comment:
Why a blank line around this string, and not the following?

Reply:
[PEP257](https://www.python.org/dev/peps/pep-0257/) puts the closing quotes on their own lines.  PyCharm defaults to inserting and extra line above - and since I think it also reads better that way, I left it.

Reply:
I'll normalize it.